DS3231.cpp

```diff
@@ -279,7 +279,7 @@ void DS3231::setEpoch(time_t epoch, bool flag_localtime) {
 	setDoW(tmnow.tm_wday + 1);
 	setDate(tmnow.tm_mday);
 	setMonth(tmnow.tm_mon + 1);
-	setYear(tmnow.tm_year - 100);
```

Comment:
Yes this should be Corrected if the Offset from 1970 to 2000 is consinderd in the seconds.

Reply:
Yes, so now it does not have any "-100" and I edit the seconds. Does this look correct to you?